modules/utils/2.1/config/default.yaml

```diff
@@ -13,7 +13,7 @@ lcr-modules:
 ## Lines commented out with `#?` can optionally be user-configured
 ## Lines commented out with `##` act as regular comments
 
-        paired_modules: ["bwa_mem", "star"]
+        paired_modules: ["bwa_mem", "star", minimap2]
```

Comment:
The utils is not imported here, should we need this modification? Does it work without quotation marks?